main.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_kinesis_stream" "kinesis_stream" {
+  name             = "${var.project}-${var.name}-${var.environment}"
+  shard_count      = "${var.kinesis_chard_count}"
+  retention_period = "${var.kinesis_retention_period}"
+
+  tags = "${var.tags}"
```

Comment:
Shouldn't we add some default tag?

Reply:
I don't see anything specific to add. Any proposal?

Reply:
by doing it like this we allow the user to set its own tags to the resource

Reply:
Nope it was just an idea but I am not sure what default tag can be added